elements/extensions/icon-theme-endless-os.bst

```diff
@@ -0,0 +1,35 @@
+kind: autotools
+
+build-depends:
+- freedesktop-sdk.bst:public-stacks/buildsystem-autotools.bst
+- gnome-sdk-sdk/gtk+-3.bst
+- gnome-sdk.bst:core-deps/intltool.bst
+- gnome-sdk.bst:sdk/appstream-glib.bst
+- gnome-sdk.bst:sdk/glib.bst
+- gnome-sdk.bst:sdk/librsvg.bst
+
```

Comment:
I think you should at least add a runtime dependency on Gtk3, otherwise there is a risk this integration command gets run **before** staging the icon theme when composing the final Flatpak image:

https://gitlab.gnome.org/GNOME/gnome-build-meta/blob/gnome-3-34/elements/sdk/gtk+-3.bst#L48-54

You can either add the following:

```
runtime-depends:
- gnome-sdk-sdk/gtk+-3.bst
```

Or you can do the following change:

```diff
 build-depends:
 - freedesktop-sdk.bst:public-stacks/buildsystem-autotools.bst
-- gnome-sdk-sdk/gtk+-3.bst
 - gnome-sdk.bst:core-deps/intltool.bst
 - gnome-sdk.bst:sdk/appstream-glib.bst
 - gnome-sdk.bst:sdk/glib.bst
 - gnome-sdk.bst:sdk/librsvg.bst

+depends:
+- gnome-sdk-sdk/gtk+-3.bst
+
```

Reply:
So if I do that the extension includes a bunch of gtk+-3 related stuff under etc (so, /usr/share/runtime/share/icons/EndlessOS/etc/), although we can strip that out if need be. I don't quite understand what this solves. Would it help if we do our own gtk-update-icon-cache here?

Reply:
> I don't quite understand what this solves.

You need to have the icon theme present in the root before running `gtk-update-icon-cache`, and I **think** you can only guarantee that if the icon theme `depends` on Gtk.

However, it seems I was wrong because in FDO Sdk Hicolor doesn't depend on Gtk: https://gitlab.com/freedesktop-sdk/freedesktop-sdk/blob/master/elements/components/hicolor-icon-theme.bst